src/nwsapi.js

```diff
@@ -91,7 +91,7 @@
     disp_state: '(open|closed|modal|fullscreen|picture-in-picture)\\b',
     time_state: '(current|past|future)\\b',
     // pseudo-classes for parsing only selectors
-    pseudo_nop: '(autofill|-webkit\\-autofill)\\b',
+    pseudo_nop: '(autofill|-webkit\\-autofill|-moz\\-read\\-only)\\b',
```

Comment:
hearing reports from my team that `-moz-read-only` was not the only offending selector observed, so possible that we'll need something more akin to the below, matching the wildcard `-webkit-*` regex defined in `pseudo_dbl` 🤔 - open to your thoughts/preferences @dperini!
```suggestion
    pseudo_nop: '(autofill|-webkit\\-autofill|-moz-[-a-zA-Z0-9]{2,})\\b',
```